libraries/shared/factories/event/README.md

```diff
@@ -149,23 +138,23 @@ type ExampleModel struct {
 }
 ```
 
-### Converter
+### Transformer
 
-The converter needs to satisfy the interface. One for unpacking logs into the custom defined entities, and
+The transformer needs to satisfy the interface. One for unpacking logs into the custom defined entities, and
 another for converting those entities to their final db models.
 
 ```go
-type Converter interface {
+type Transformer interface {
 	ToEntities(contractAbi string, ethLog []types.Log) ([]interface{}, error)
 	ToModels([]interface{}) ([]interface{}, error)
 }
 ```
 
 For the example event, this might look like:
 ```go
-type ExampleConverter struct{}
+type ExampleTransformer struct{}
 
-func (ExampleConverter) ToEntities(contractAbi string, ethLogs []types.Log) ([]interface{}, error) {
+func (ExampleTransformer) ToEntities(contractAbi string, ethLogs []types.Log) ([]interface{}, error) {
```

Comment:
To Elizabeth's point ToEntities is not part of the interface - and typically private isn't it?